src/Infrastructure/Service/Filesystem.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Sdk\Infrastructure\Service;
+
+use Symfony\Component\Filesystem\Filesystem as SymfonyFilesystem;
+
+class Filesystem extends SymfonyFilesystem implements FilesystemInitInterface
```

Comment:
Would be fine also to add FilesystemInterface

Reply:
For what?

Reply:
Need to abstraction to the file system due to OCP, to have ability to use own fs (not only symfony). Let make it in separate task